include/engine/guidance/leg_geometry.hpp

```diff
@@ -38,8 +38,12 @@ struct LegGeometry
     struct Annotation
     {
         double distance; // distance in meters
-        double duration; // duration in seconds
-        double weight;   // weight value
+
+        // Total duration of a segment, in seconds, NOT including
+        // the turn penalty if the segment preceeds a turn
+        double duration;
+        double weight; // weight value, including the turn weight
```

Comment:
I can see how it makes sense to not report the turn duration in the annotation - but why do we then report the turn weight in the annotations? Can and should we at least make the duration and weight annotation _both_ not report the turn duration and weight, respectively?